man/fail2ban-testcases.1

```diff
@@ -0,0 +1,41 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.46.4.
+.TH FAIL2BAN-TESTCASES "1" "February 2015" "fail2ban-testcases 0.9.1" "User Commands"
+.SH NAME
+fail2ban-testcases \- manual page for fail2ban-testcases 0.9.1
```

Comment:
mneh -- have a look at   man/generate-man ;-) just add a corresponding stuff for the fail2ban-testcases ;-)